src/test/resources/uk/gov/ons/ctp/integration/rhsvc/service/impl/CaseServiceImplTest.CaseDTO.json

```diff
@@ -1,9 +1,6 @@
 [
 	{
 		"caseId": "dc4477d1-dd3f-4c69-b181-7ff725dc9fa4",
-		"caseRef": "10000000010",
```

Comment:
This JSON is the wrong shape, and doesn't seem to be used. Can probably be deleted.